tasks.py

```diff
@@ -355,6 +384,8 @@ def build_and_check_docs(context):
 )
 def unittest(context, keepdb=False, label="nautobot_chatops", failfast=False, buffer=True):
     """Run Nautobot unit tests."""
+    docker_compose(context, "rm --stop -- nautobot")
```

Comment:
Let's be sure to add a comment about why this is in here.

Reply:
This line was removed as I removed Bootstrap plugin. With the current Nautobot implementation, it's not possible to have multiple Apps in a single package: https://github.com/nautobot/nautobot/blob/bdf1faa6386a0fa866978f6eb866413a336cb9f9/nautobot/extras/plugins/urls.py#L27